tic-tac-toe-python/Player.py

```diff
@@ -0,0 +1,15 @@
+class Player:
+
+    def __init__(self,playerSymbol):
+        self.symbol=playerSymbol
+
+    def validateCoordinates(self,x,y):
```

Comment:
why validatecoordinates is in Player? should not this be in Game ?

Reply:
There are two types of validations one is in the game (which checks if the cell is occupied) and the other in player (which just checks if the coordinates are in range [1,1] to [3,3]. For this we dont need to actually go to the gameBoard).

Reply:
What if the game board is bigger? @nameetj3416

Reply:
Ohkay so this [3,3] thing should be configurable right ? (But does it make sense as far as the scope of Tic-Tac-Toe is concerned)